src/core/public/styles/rendering/_base.scss

```diff
@@ -75,9 +75,6 @@
   &.kbnBody--chromeHidden {
     @include kbnAffordForHeader(0);
   }
-  &.kbnBody--projectLayout {
-    @include kbnAffordForHeader($euiHeaderHeightCompensation);
-  }
```

Comment:
No longer needed since we're using a constant 2 header bars in the project layout